gnat2goto/driver/goto_utils.adb

```diff
@@ -121,6 +121,19 @@ package body GOTO_Utils is
       return Ret;
    end Symbol_Expr;
 
+   procedure New_Type_Symbol_Entry (Type_Name : String; Type_Type : Irep;
```

Comment:
`Type_Type` type type type type.

Reply:
Well before it was `actual_type`. I can re-name it back.